internal/schema/stacks/1.8/variable_block.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/tokmod"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func variableBlockSchema() *schema.BlockSchema {
+	return &schema.BlockSchema{
+		SemanticTokenModifiers: lang.SemanticTokenModifiers{tokmod.Variable},
+		Labels: []*schema.LabelSchema{
+			{
+				Name:                   "name",
+				SemanticTokenModifiers: lang.SemanticTokenModifiers{tokmod.Name},
+				Description:            lang.PlainText("Variable Name"),
+			},
+		},
+		Description: lang.Markdown("Stack variable allowing users to customize aspects of the stack that differ between deployments" +
+			"(e.g. different instance sizes, regions, etc.)"),
```

Comment:
self review: I changed this wording, but am a bit unsure whether it's too prescriptive. What do you think, @jpogran?

Reply:
I appreciate the wordsmithing, but do not think things are solidified yet to gate things on wording that will most likely expand or change in the near future. LGTM now.